auth/template.yaml

```diff
@@ -452,11 +452,46 @@ Resources:
         Environment: !Ref Environment
         System: Authentication
 
+  CognitoClientSecret:
+    Type: AWS::SecretsManager::Secret
+    Properties:
+      Name: /cognito/client-secret
+      Description: Cognito App Client Secret
+      # not setting secret string avoids being overridden on subsequent deployments
```

Comment:
I've performed multiple sam deploy's and it hasn't caused the secret value to be overwritten